nvbios/nvbios.c

```diff
@@ -605,8 +605,21 @@ void printscript (uint16_t soff) {
 				}
 				break;
 			case 0x8c:
+				/*
+				 * Signal that the reset sequence has begun.
+				 *
+				 * This opcode signals that the software reset sequence has begun.
+				 * Ordinarily, no actual operations are performed by the opcode.
+				 * However it allows for possible software work arounds by devinit
+				 * engines in software agents other than the VBIOS, such as the resman,
+				 * FCODE, and EFI driver. This opcode is designed to be included in
+				 * the devinit script immediately after the NV_PMC_ENABLE register
+				 * has be written to put most engines into a reset state.
+				 *
+				 * Present on [core6,)
```

Comment:
Is this a typo?

Reply:
I was using this notation to describe the interval (e.g. *inclusive* of core6 on the lower bound, presently unbounded at the top end).

Reply:
Discussed and resolved on IRC.